src/data/sql/hr/create.functievervulling.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE hr_prep.functievervulling AS
+
+  SELECT
+    fvv.ashid::varchar                                          AS identificatie,
+    NULL::boolean                                               AS langstzittende,
+    NULL::date                                                  AS datum_aanvang,
+    NULL::date                                                  AS datum_einde,
+    NULL::varchar                                               AS functie_titel,
+    NULL::boolean                                               AS indicatie_statutair,
+    NULL::jsonb                                                 AS schorsing,
+
+    CASE -- aansprakelijke
+      WHEN fvv.functie IS NULL THEN NULL::jsonb
+      ELSE JSONB_BUILD_OBJECT(
+        'code', NULL,
+        'omschrijving', fvv.functie
+      )
+    END                                                         AS aansprakelijke,
+    NULL::varchar                                               AS handelingsbekwaam,
+
+    CASE -- bevoegdheids_aansprakelijke
+      WHEN fvv.soort IS NULL THEN NULL::jsonb
+      ELSE JSONB_BUILD_OBJECT(
+        'code', NULL,
+        'omschrijving', fvv.soort
+      )
+    END                                                         AS bevoegdheids_aansprakelijke,
+
+    NULL::jsonb                                                 AS bestuursfunctie,
+    NULL::jsonb                                                 AS bevoegdheid_bestuurder,
+    NULL::varchar                                               AS vertegenwoordiger_bestuurder_rechtspersoon,
+    NULL::jsonb                                                 AS gemachtigde,
+    NULL::boolean                                               AS volmacht,
+    NULL::boolean                                               AS statutair,
+    NULL::jsonb                                                 AS heeft_h_rvestiging, -- relaties VES
```

Comment:
Dient deze niet de type `json []` hebben? staat als een array van json's in Amsterdam schema gespecificeerd.

Reply:
Klopt, maar dat is alsnog JSONb data, type `jsonb` in Postgres.